.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,77 @@
+name: Build Wheels
+
+# Todo: remove push
+on: [workflow_dispatch, push]
+
+jobs:
+  macos:
+    name: macOS Wheels
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+          architecture: x64
+      - uses: dtolnay/rust-toolchain@stable
+      - name: Build wheels - universal2
+        uses: PyO3/maturin-action@v1
+        with:
+          args: --release --universal2 --out dist -m minijinja-py/Cargo.toml
+      - name: Upload wheels
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheels
+          path: dist
+
+  linux:
+    name: Linux Wheels
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: [aarch64, armv7, s390x, ppc64le, ppc64]
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+          architecture: ${{ matrix.target }}
```

Comment:
GitHub Actions only has x64 & x86 Python versions. 

To build manylinux compliant wheels `pyo3/maturin-action` can't use these python interpreters installed by `actions/setup-python`, it uses the official `pypa/manylinux` docker images for x86_64 and x86 targets, and for other cpu architectures by default it uses the [manylinux-cross](https://github.com/rust-cross/manylinux-cross) docker images.

In fact, maturin doesn't really need a working target Python interpreter to build wheels for Linux, all it needs is the right target Python version, `EXT_SUFFIX`, `SOABI` and pointer width  from `sysconfig`, so we extracted the related values from `sysconfig` and [shipped them with `maturin`](https://github.com/PyO3/maturin/blob/main/src/python_interpreter/sysconfig-linux.json).

For example to cross compile a Linux aarch64 wheel for Python 3.10, `maturin build --target aarch64-unknown-linux-gnu --interpreter python3.10` should do as long as the build environment has the appropriate version of GCC for linking and compiling C/C++ dependencies, which are provided by `pypa/manylinux` and `manylinux-cross` projects.

```yaml
      - name: Build wheels - universal2
        uses: PyO3/maturin-action@v1
        with:
          target: ${{ matrix.target }}
          args: --release --out dist -m minijinja-py/Cargo.toml --interpreter python3.10
          manylinux: auto
```

The above example should work for building Python 3.10 wheels.

Reply:
Yeah I know it doesn't have it. Sadly whenever I set up this stuff it's suffering from scratch.

Reply:
2a42d9b359d47023f3c7d565756045540a0292d3 was pretty close to work, I think the issue is that maturin doesn't ship sysconfig for the `ppc64` architecture. `ppc64` was unsupported since manylinux_2_24 and major Linux distros have dropped support.